docs/feature-flagging/qa-flags.md

```diff
@@ -0,0 +1,29 @@
+# QA Flags
+
+## Testing Environments
+
+Eppo supports separation of feature flags and their allocations between environments. Create an Eppo environment specifically for Staging or QA following this [documentation](/sdks/api-keys). Environments are separated by [SDK keys](/sdks/api-keys), and no unique key will be shared across Eppo Environments. Once an Environment is created, create an SDK for that Environment.
```

Comment:
/feature-flags/index#environments is a better place to link to. It has all this information

Reply:
fixed